mesprojets.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
"fr".  Also, try to use file names in English, while your content can be in French.

Reply:
Done.